src/Microsoft.Health.Fhir.Shared.Api.UnitTests/Features/Audit/AuditEventTypeMappingTests.cs

```diff
@@ -11,8 +11,8 @@
 using Microsoft.AspNetCore.Mvc.Controllers;
 using Microsoft.AspNetCore.Mvc.Infrastructure;
 using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Health.Api.Features.Audit;
```

Comment:
These tests should move together with the code right?

Reply:
Yep, I will move them. Quick question, should we also keep them here in FHIR or just having them in the shared components would suffice?

Reply:
We only need these in the shared. FHIR has its own integration tests that validates the functionality already so we don't need these unit tests here.

Reply:
Makes sense. I will delete these tests from here. I have already pushed a new iteration to the other pull request (in healthcare shared components) with these tests added.